index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Redressed&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body class="body-class">
+
+    <h1 id="title">freeCodeCamp Survey Form </h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p><br> 
```

Comment:
I think it is better to use HTML5 semantic tag. (HTML5 semantic tag makes your code cleaner and better to read in the search engine).